src/Umbraco.Web/Security/Providers/UmbracoMembershipProvider.cs

```diff
@@ -450,22 +456,15 @@ protected override string PerformResetPassword(string username, string answer, s
         /// </returns>
         public override bool UnlockUser(string username)
         {
-            var member = MemberService.GetByUsername(username);
-
-            if (member == null)
-            {
-                throw new ProviderException(string.Format("No member with the username '{0}' found", username));
-            }                
-
-            // Non need to update
-            if (member.IsLockedOut == false) return true;
-
-            member.IsLockedOut = false;
-            member.FailedPasswordAttempts = 0;
-
-            MemberService.Save(member);
+            var userManager = GetBackofficeUserManager();
```

Comment:
Ack, this will mean this method no longer works at all for members. I'll revert this and to continue using this it needs to be overridden in the `UserMembershipProvider`